hw03_michaelh.py

```diff
@@ -0,0 +1,151 @@
+
+
+#################################################################################
+# Author: Hope Michael
+# Username: michaelh
+#
+# Assignment: HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose: Draw a house with a tree next to it
+# Google Doc Link: https://docs.google.com/document/d/1lhv3-v5w_d1k4s5Q-C-Qu92dv-KYypOBRtJEx8BTQ8A/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+# Sami (Partner)
+#
+#################################################################################
+
+
+import turtle
+
+
+def make_turtle(size, color,shape):
+    """
+    makes a turtle
+    :param size: size of the turtle
+    :param color: color of the turtle
+    :param shape: shape of the turtle
+    """
+    t = turtle.Turtle()
+    t.hideturtle()
+    t.color(color)
+    t.shape(shape)
+    t.pensize(size)
+    t.speed(0)
+    return t
+
+def setpos(turtle, x, y):
```

Comment:
It is generally a bad practice to ever name a turtle `turtle` because it collides with the `turtle` module name, and causes bad things to happen.